lib/build.gradle

```
@@ -20,7 +20,7 @@ plugins {
 }
 
 group 'com.dynatrace.metric.util'
-version = '0.2.1'
+version = '0.2.2'
```

We're in 0.x anyway but technically adding a further case where the MetricException is thrown would be considered a breaking change (although it's a checked exception anyway already), so we should bump to 0.3.

```suggestion
version = '0.3.0'
```